### ISSUE TITLE
Types clash when using along with @ionic/core@4.1.1 (again 🙄)

### ISSUE THREAD

**Issue by simonhaenisch** [ionic-team/ionic-pwa-elements#18]
@mlynch this is getting quite frustrating to work with... every time an Ionic interface changes, the `@ionic/pwa-elements` package is blocking us from updating `ionic/core` in our project.

```
[ ERROR ]  TypeScript: node_modules/@ionic/pwa-elements/node_modules/@ionic/core/dist/types/components.d.ts:5176:4
           Subsequent property declarations must have the same type. Property ''ion-checkbox'' must be of type
           'IonCheckboxAttributes', but here has type 'IonCheckboxAttributes'.

   L5175:  'ion-card': Components.IonCardAttributes;
   L5176:  'ion-checkbox': Components.IonCheckboxAttributes;
   L5177:  'ion-chip': Components.IonChipAttributes;
```

etc.

I can confirm that updating `@ionic/core` to `4.1.1` within `@ionic/pwa-elements` fixes the issue.

I was thinking "just make `"@ionic/core": "^4.0.0"` a peer dependency", but then I saw that the `camera-modal` component imports the `Modal` interface... so I guess it's not that simple.

---

I tried as a work-around to just use the script tag method, i. e. add a copy task to my stencil config and a script tag to index.html, however that doesn't work either, throws at runtime:

```
ionic config was already initialized
    setupConfig | @ | app.core.js:6
```

```
app.core.js:146 Uncaught TypeError: Cannot redefine property: queue
    at Function.defineProperty (<anonymous>)
    at app.core.js:146
```

---

TL;DR we need this package to be kept up to date.
